Config.uk

```diff
@@ -180,6 +180,12 @@ if RECORD_BUILDTIME
 	endchoice
 endif
 
+config BUILD_ARGS_FILE
```

Comment:
Hum... but since the buildrule is modified, if something would break, it would have broke already or should be actually tested. Every C/C++ etc. file is going through the rules that you modiifed.

Reply:
The buildrule I modified in PR #619 is only for the linking of the preolibs, it could (in theory) break if somebody uses a linker that does not support reading the arguments from a file. Otherwise I don't see a problem, and even this will go away if we just write out the whole command :+1:
The build rules for C/C++ in this PR are more complicated, this is why I split this from the PR #619.

So in short: I have no problem with removing config option.

Reply:
Okay, I think we should remove the config option, I am concerned that this option is confusing and a weird workaround to fix some cases (if it stays).